src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/EventsController.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Repository;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Altinn.Platform.Events.Controllers
+{
+    /// <summary>
+    /// Provides operations for handling events
+    /// </summary>
+    [Route("events/api/v1/events")]
+    [ApiController]
+    public class EventsController : ControllerBase
+    {
+        private readonly IEventsRepository repository;
+        private readonly ILogger logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EventsController"/> class
+        /// </summary>
+        /// <param name="repository">the events repository handler</param>
+        /// <param name="logger">dependency injection of logger</param>
+        public EventsController(IEventsRepository repository, ILogger<EventsController> logger)
+        {
+            this.repository = repository;
+            this.logger = logger;
+        }
+
+        /// <summary>
+        /// Inserts a new event.
+        /// </summary>
+        /// <param name="cloudEvent">The event to store.</param>
+        /// <returns>The applicaiton metadata object.</returns>
+        // [Authorize(Policy = AuthzConstants.POLICY_STUDIO_DESIGNER)]
```

Comment:
The commented line with the authorize tag can be removed for now I guess.

Reply:
Done :)